src/mpi/coll/red_scat/red_scat.c

```diff
@@ -12,16 +12,45 @@
 === BEGIN_MPI_T_CVAR_INFO_BLOCK ===
 
 cvars:
-    - name        : MPIR_CVAR_REDSCAT_COMMUTATIVE_LONG_MSG_SIZE
```

Comment:
Why did this CVAR go away?

Reply:
@wesbland this should be there and will insert it back